consensus/misc/eip1559_test.go

```diff
@@ -45,6 +45,7 @@ func copyConfig(original *params.ChainConfig) *params.ChainConfig {
 		BerlinBlock:             original.BerlinBlock,
 		LondonBlock:             original.LondonBlock,
 		TerminalTotalDifficulty: original.TerminalTotalDifficulty,
+		CancunBlock:             original.CancunBlock,
```

Comment:
@jwasinger do we need to add this? It's a test, and imo the value should be `nil` here.

Reply:
Sure, we can omit it.